Deploy/stacks/dynamic/examples/datasets/inputs/config/building-cambridge.json

```diff
@@ -25,11 +25,11 @@
             "importOptions": {
                 "sridIn": "27700"
             },
-            "name": "cambridge_gsl",
+            "name": "building-cambridge_gsl",
             "geoServerSettings": {
                 "virtualTable": {
                     "name": "mapbox_layer",
-                    "sql": "@/inputs/config/building-cambridge.sql",
+                    "sql": "@/inputs/config/mapbox-footprint.sql",
```

Comment:
it's not clear at first that this file needs be copied. Also a general comment that the example folder is getting crowded with the growing number of examples

Reply:
It is explained in the README
https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-data-uploader/README.md#value-by-file-name